FUNCTIONS/calculator.py

```diff
@@ -0,0 +1,39 @@
+def add_number(number_1: int, number_2: int) -> int:
+    return number_1 + number_2
+
+
+def substract_number(number_1: int, number_2: int) -> int:
+    return number_1 - number_2
+
+
+def multiply_number(number_1: int, number_2: int) -> int:
+    return number_1 * number_2
+
+
+def divide_number(number_1: int, number_2: int) -> float:
```

Comment:
Why int everywhere but here float?

Reply:
Because https://docs.python.org/3/tutorial/introduction.html#numbers
![image](https://github.com/MarinaZelenska/test_repo_python/assets/51053483/c5a3e7fb-36d9-4d00-9427-6d8862a945b8)